src/libraries/Common/src/System/Obsoletions.cs

```diff
@@ -190,6 +189,9 @@ internal static class Obsoletions
         internal const string SystemEventsEventsThreadShutdownMessage = "SystemEvents.EventsThreadShutdown callbacks are not run before the process exits. Use AppDomain.ProcessExit instead.";
         internal const string SystemEventsEventsThreadShutdownDiagId = "SYSLIB0059";
 
+        internal const string Rfc2898DeriveBytesCtorMessage = "Creating instances of Rfc2898DeriveBytes is obsolete. Use the static Pbkdf2 method instead.";
```

Comment:
The wording of this is a little strange to my ears; it's odd that "creating" would be obsolete. Could we instead say "The constructors on Rfc2898DeriveBytes are obsolete"?